.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add this file to your .gitignore file, this file should not be committed, as it would be  different for everyone, and will cause problems for others.
Read about .gitignore, you list fileNames/folders/regex to exclude files from git. Such files will stay in your local, and won't be pushed to github.
@PARIMALPRATYUSH  @Girrajgg @pallav2899  -> Pay attention to this.

Reply:
You should add .idea folder to gitignore